carrot.css

```
@@ -0,0 +1,141 @@
+.woo-part2-1-background {
+  background-color: #e6f3e6;
+  width: 100%;
+  justify-content: space-evenly;
+}
+
+.woo-part2-1-background .woo-content-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: #e6f3e6;
+  width: 1024px;
+  height: 740px;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+.woo-part2-1-background .woo-title-neiborhood {
+  display: flex;
+  flex-direction: column;
+}
+
+.woo-title-icon {
+  display: flex;
+  flex-direction: row;
+  margin-top: 40px;
+  display: flex;
+  justify-content: space-between;
+  font-weight: 400;
+  width: 478px;
+  height: 172px;
+}
+
+.woo-title-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-right: 40px;
+  margin-left: 0;
+  width: 140px;
+  max-width: 140px;
+}
+
+.woo-title-buttons-one {
+  display: flex;
+  flex-direction: column;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-right: 0px;
+  margin-left: 0;
+  max-width: 140px;
+  width: 140px;
+  height: 140px;
+}
```

어떤 연속된 개체들 사이의 부분에만 공간을 주고 싶다면, 각 개체들에 margin-right 또는 margin-left 속성을 주는 것보다, 그것을 감싸고 있는 flex container에 gap 속성을 주면 쉽게 해결할 수 있습니다.

gap 속성은 flex container 안에 있는 개체들의 사이에만 빈 공간을 만들어줍니다.

```suggestion
.woo-title-icon {
  display: flex;
  flex-direction: row;
  margin-top: 40px;
  display: flex;
  gap: 40px; /* 추가하기 */
  justify-content: space-between;
  font-weight: 400;
  width: 478px;
  height: 172px;
}

.woo-title-buttons {
  display: flex;
  flex-direction: column;
  margin-top: 16px;
  margin-bottom: 16px;
  /* margin-right: 40px; 없애도 됩니다! */
  /* margin-left: 0; 이것도 의미가 없으니 없애도 되겠죠 */
  width: 140px;
  max-width: 140px;
}
```

이렇게 기존 클래스를 지웠을 경우, .woo-title-buttons-one 클래스 네임을 적용시켰던 div에 .woo-title-buttons를 대신 적용해줘야겠죠!